commands/scs_dice.py

```diff
@@ -360,20 +360,44 @@ class CmdEZRoll(default_cmds.MuxCommand):
     # Uses yield() to let the player input the string components step by step.
     # Once it works, make sure it can parse for roll/call as well. This'll set up nicely for the next stage,
     # when roll/call is compatible with the combat system and can, e.g., inflict debuffs. (roll/stake, roll/gm for both)
-    # TODO: add proper docstring
     """
-    EZRoll command
+    EZRoll allows for the step-by-step assembly of a dice string to assist with
+    complex rolls. For example, using roll, a 1d20 with a +2 modifier trying to
+    beat a target number of 10 or higher with a comment of "Dodging goblin"
+    would be:
+
+    > roll 1d20+2>=10#Dodging goblin
+
+    Typing "ezroll" or "ez" instead prompts you to input each element one by one.
+    You may go back a step with "u" or quit at any time with "q". It is also
+    compatible with roll/call, which sends roll requests to other players.
+
+    Examples:
+        ez
+        ez/call reize, ivo
     """
     key = "ezroll"
     aliases = ["ez"]
     locks = "cmd:all()"
 
     def func(self):
-        caller = self.caller
-        # TODO: check for switches and integrate roll/call (need to check for args representing targets
-        # Embed the yield calls in a while loop and set the condition for the while loop such that it depends on how
+        # Embeds the yield calls in a while loop and sets the condition for the while loop such that it depends on how
         # many steps will be in the roll string. As I add more options, like "staking" buffs/debuffs on the roll result,
         # I can modify max_steps based on self.switches so the player can specify how complex the roll will be.
+        caller = self.caller
+        switches = self.switches
```

Comment:
more food for thought than a recommendation here: it may or may not be necessary to create the `args` and `switches` variables here since it looks like each is only referenced twice. It's definitely more of a preferential thing than anything else since you'd literally be saving like...single bytes of space